frontend/src/pages/Admin/__tests__/CourseAdd.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {
+  render,
+  screen,
+} from '@testing-library/react';
+import fetchMock from 'fetch-mock';
+import CourseAdd from '../CourseAdd';
+
+describe('CourseAdd', () => {
+  const mockRefresh = jest.fn();
+
+  beforeEach(() => {
+    fetchMock.reset();
+    render(<CourseAdd refresh={mockRefresh} />);
+  });
+
+  it('renders the CourseAdd component', () => {
```

Comment:
For coverage do you have a check for the ' Add course' button click?

Reply:
Just added this: 841e49359